mlrun/api/api/endpoints/functions.py

```diff
@@ -160,12 +161,44 @@ def build_status(
     tag: str = "",
     offset: int = 0,
     logs: bool = True,
+    last_time: int = 0,
```

Comment:
`last_time` is a very bad naming, only when I got inside nuclio-jupyter code I understood what does it mean - the timestamp from which you want to get logs (making sure I understood correctly ?)
Please rename it (at least in MLRun) to something more clear:
`logs_since_timestamp` / `logs_start_at_timestamp` or something similar

Reply:
changed to `last_log_timestamp`

Reply:
You're thinking too much in the context of the code you're writing in the client - that want to get logs from the timestamp of the last log
But you're creating an API here, that may be used by users, think of the context of the user, what is this variable ? just the since logs timestamp - just like k8s, they calling the variable just `since`, I would do it as well but:
1. this is not API just for logs, so adding the `logs` is a must so it will be clear this is affecting the logs
2. k8s `since` accept stuff like `1s` i.e. intervals, while we accept only timestamp
therefore `logs_since_timestamp` seems much more appropriate IMO

Reply:
i dont agree, the timestamp is the one for the last log, which is exactly what i named it, its not `since` (duration)